.ci/setup.sh

```diff
@@ -102,6 +102,11 @@ install_extra_tools() {
 		source "${cidir}/${arch}/lib_setup_${arch}.sh"
 	fi
 
+	# Do not install inmediately kubernetes on Fedora
```

Comment:
Typo: "immediately". Can you explain why k8s isn't installed immediately though?